tests/e2e/specs/swap-tokens.spec.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable ui-testing/no-disabled-tests */
```

Comment:
for consistency, should we use `test` folder instead of `tests`? I am using `test` as the folder name in wallet app and inter dapp.

Reply:
i think tests is ok if the folder doesnt already exist, otherwise we'd have to change a lot of the config